org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/RuntimeProjectDescriptor.xtend

```diff
@@ -280,10 +279,10 @@ class RuntimeProjectDescriptor extends TestedProjectDescriptor {
 				}
 
 				dependencies {
-					mwe2 "org.eclipse.emf:org.eclipse.emf.mwe2.launch:�config.xtextVersion.mweVersion�"
```

Comment:
should the values in the config then be marked as deprecated or are they used elsewhere?

Reply:
good point. I'll check that. Likely they can be deprecated.

Reply:
It does not work to remove dependency versions from exec-maven-plugin :( So there versions stay mandatory.

Reply:
But we should pin xtext-antlr-generator to 2.1.1 and no open range. There is no other version and won't.

Reply:
:+1: